pkg/gql/fetch.go

```diff
@@ -475,11 +466,11 @@ func isBlacklisted(user string) bool {
 
 // setupProgressBar sets the progress bar properly according to
 // the expected amount of pages of data.
-func setupProgressBar(pages int) *mpb.Bar {
+func setupProgressBar(pages int) (*mpb.Progress, *mpb.Bar) {
 	p := mpb.New(mpb.WithWidth(64))
 
 	bar := p.AddBar(int64(pages*contribPagination),
-		mpb.BarStyle("[=>-]"),
+		mpb.BarRemoveOnComplete(),
```

Comment:
BarRemoveOnComplete is here to preserve original visual behavior, however if you'd like to keep bar it's ok to remove this option.